kuma/static/js/wiki.js

```diff
@@ -148,7 +154,7 @@
     /*
         Syntax highlighting scripts
     */
-    if($('article pre').length && ('querySelectorAll' in doc)) (function() {
+    if($('article pre').length && ('querySelectorAll' in doc)) {
```

Comment:
I'm not sure if I did the right thing here. Should I have preserved the `(function(){})();`